src/GraphManager/utils.test.ts

```diff
@@ -264,3 +264,15 @@ describe("linkPointerAreaPaint", () => {
     expect(ctx.stroke).not.toHaveBeenCalled();
   });
 });
+
+
+describe("onGraphUpdate", () => {
+    it("should update the graph data", () => {
+        const ctrl = makeMockController();
+        const graphData = { nodes: [], links: [] };
+        makeGraphState(graphData, jest.fn() );
+        // SHOULD BE
+        // onGraphUpdate(ctrl, graphData) instead
```

Comment:
Here you should create the `const setGraph = jest.fn()` as a mock function and pass it to `onGraphUpdate(ctrl, graphData, setGraph);`
Then you can check for a call to the jest.fn, see https://jestjs.io/docs/mock-functions (probably something like `expect(setGraph).toHavebeenCalledWith( the graph state )`.